borgbase/vorta
# Implement add profile button (#158)

**Background:**
Currently the add profile action is inside of the profile dropdown list
as `+ Add profile` while edit and delete have their own buttons. This
commit moves the add profile action out of the dropdown into its own
button for UI consistency.

**Modifications:**
* Update UI of `mainwindow.ui` to add a new add profile button.
* Add fontawesome `plus.svg` icon to both light and dark icon sets and
rerun `pyrcc5`.
* Update `main_window.py` to wire up the new add profile button.

**Steps to test:**
1. Launch `vorta` from the command line.
2. Click + and verify that it opens the add profile window.
3. Add a new profile and see that it gets added.
4. Delete the profile that was just added.
5. Click + but cancel the add profile action. Verify that nothing
happens and the current profile stays selected.

Fixes https://github.com/borgbase/vorta/issues/158

<img width="800" alt="screen shot 2019-02-03 at 7 59 21 am" src="https://user-images.githubusercontent.com/1526881/52179106-fb80c080-278a-11e9-90b8-3c05a1de3bca.png">
<img width="786" alt="screen shot 2019-02-03 at 8 00 23 am" src="https://user-images.githubusercontent.com/1526881/52179107-fe7bb100-278a-11e9-882f-c08fc0aa9a60.png">
